widget/embedded/src/components/Layout/Layout.styles.ts

```diff
@@ -8,6 +8,7 @@ export const Container = styled('div', {
   overflow: 'hidden !important',
   boxShadow: '15px 15px 15px 0px rgba(0, 0, 0, 0.05)',
   width: '95vw',
+  minWidth: '300px',
```

Comment:
Can you check these cases?

![Screenshot from 2023-12-11 14-00-45](https://github.com/rango-exchange/rango-client/assets/99180934/b6ca7375-51a2-4240-8076-34d01c809a34)
![Screenshot from 2023-12-11 14-00-52](https://github.com/rango-exchange/rango-client/assets/99180934/e11513af-0660-4f16-a995-b53823d392b4)
![Screenshot from 2023-12-11 14-08-10](https://github.com/rango-exchange/rango-client/assets/99180934/014c4a68-ec7b-4f4c-8362-ee6727dda4ab)
![Screenshot from 2023-12-11 14-12-27](https://github.com/rango-exchange/rango-client/assets/99180934/fb186e72-d644-43b2-bc4e-f25a59fb54db)